src/meson.build

```diff
@@ -18,7 +18,7 @@ libnetplan = library(
     gnu_symbol_visibility: 'hidden',
     dependencies: [glib, gio, yaml, uuid],
     include_directories: inc,
-    soversion: '0.0',
+    soversion: 1,
     install: true)
 
 libnetplan_testing = library(
```

Comment:
nitpick (non-blocking): `soversion` should be changed in `libnetplan_testing` as well for consistency.